prime-router/src/main/kotlin/transport/BlobStoreTransport.kt

```diff
@@ -26,7 +26,7 @@ class BlobStoreTransport() : ITransport {
             val receiver = header.receiver ?: error("No receiver defined for report ${header.reportFile.reportId}")
             val bodyUrl = header.reportFile.bodyUrl ?: error("Report ${header.reportFile.reportId} has no blob to copy")
             context.logger.info("About to copy $bodyUrl to $envVar:$storageName")
-            var newUrl = BlobAccess.copyBlob(bodyUrl, envVar, storageName)
+            val newUrl = BlobAccess.copyBlob(bodyUrl, BlobAccess.BlobContainerMetadata.build(transportType))
```

Comment:
This is currently the only example of alternate environment use - in this case, an environment external to RS when a receiver uses the blob store transport method.